.ci/install_qemu_lite.sh

```diff
@@ -16,33 +16,28 @@
 
 set -e
 
-if [ "$#" -ne 3 ]; then
-    echo "Usage: $0 <CLEAR_RELEASE> <QEMU_LITE_VERSION> <DISTRO>"
-    echo "       Install the QEMU_LITE_VERSION from clear CLEAR_RELEASE."
-    exit 1
-fi
-
-clear_release="$1"
-qemu_lite_version="$2"
-distro="$3"
-qemu_lite_bin="qemu-lite-bin-${qemu_lite_version}.x86_64.rpm"
-qemu_lite_data="qemu-lite-data-${qemu_lite_version}.x86_64.rpm"
-
-echo -e "Install qemu-lite ${qemu_lite_version}"
-
-# download packages
-curl -LO "https://download.clearlinux.org/releases/${clear_release}/clear/x86_64/os/Packages/${qemu_lite_bin}"
-curl -LO "https://download.clearlinux.org/releases/${clear_release}/clear/x86_64/os/Packages/${qemu_lite_data}"
-
-# install packages
-if [ "$distro" == "fedora" ]; then
-    sudo rpm -ihv "./${qemu_lite_bin}" --nodeps
-    sudo rpm -ihv "./${qemu_lite_data}" --nodeps
-elif [ "$distro" == "ubuntu" ];  then
-    sudo alien -i "./${qemu_lite_bin}"
-    sudo alien -i "./${qemu_lite_data}"
-fi
-
-# cleanup
-rm -f "./${qemu_lite_bin}"
-rm -f "./${qemu_lite_data}"
+QEMU_REPO="qemu"
+
+git clone https://github.com/qemu/qemu
+
+pushd $QEMU_REPO
+git checkout stable-2.10
+
+curl https://raw.githubusercontent.com/clearcontainers/packaging/master/qemu-lite/configure.patch | patch -p1
+
+./configure --disable-static --disable-bluez --disable-brlapi --disable-bzip2 --disable-curl \
```

Comment:
I appreciate it would be slightly awkward to do this (as it's in a different repo), but I'd still like us to try moving to using https://github.com/clearcontainers/packaging/blob/master/scripts/configure-hypervisor.sh. Once that script is being used by tests and packaging, we have a single location for adjusting hypervisor built-time options.